README.md

```diff
@@ -1,56 +1,109 @@
+- [API Endpoints](https://agsaurabh.pythonanywhere.com/api/v1/overview/) (API Docs coming soon) <!-- @TODO: Add the Docs -->
 
 ## Local Setup
- - [Server Setup](/server/README.md)
- - [Client Setup](/client/README.md)
+
+1. Clone the project
+2. Change directory to the root of the project
+
+### Backend Setup
+
+1. Create a [Python Virtual Environment](https://www.geeksforgeeks.org/python-virtual-environment/#:~:text=A%20virtual%20environment%20is%20a,of%20the%20Python%20developers%20use.) and activate it
+
+2. Move to the [`server`](server) directory and install the dependencies using the command
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+3. Move to the [`src`](server/src) directory and execute the following commands to setup and run the project
+
+   ```bash
+   python3 manage.py makemigrations
+   python3 manage.py migrate
+   python3 manage.py collectstatic
+   python3 manage.py runserver
+   ```
+
+4. Visit [`localhost:8000`](http://localhost:8000) to see the deployed backend
+
+### Frontend Setup
+
+1. Setup [Yarn](https://yarnpkg.com/)
+
+2. Move to the [`client`](client) directory and execute the following commands to setup and start the client side app
+
+   ```bash
+   yarn install
+   yarn start
+   ```
+
+3. Visit [`localhost:3000`](http://localhost:3000) to see the deployed backend
+
+> Note: For the client to display the response of the backend endpoints, the `API endpoint` needs to be updaed in the client side code according to where it is running
 
 ## Future Scope
+
 We will be mainly focusing on adding more features to this project, but the main focus will be on these things:
- - Made data to be updated on hourly basis (or faster if possible).
- - Issue Suggestions
- - UI for top contributors
- - Map for showing country-wise contributions
- - UI for Popular Languages and Repositories
- - UI improvements
+
+- Made data to be updated on hourly basis (or faster if possible).
+- Issue Suggestions
+- UI for top contributors
+- Map for showing country-wise contributions
+- UI for Popular Languages and Repositories
+- UI improvements
 
 ## Authors
+
 | <img src="https://github.com/AlbertoPdRF.png" alt="Alberto Pérez de Rada Fiol" width="100" height="100" /> | <img src="https://github.com/LakshyaKhatri.png" alt="Lakshya Khatri" width="100" height="100" /> | <img src="https://github.com/SaurabhAgarwala.png" alt="Saurabh Agarwala" width="100" height="100" /> |
-| ----- | ----- | ----- |
-| [Alberto Pérez de Rada Fiol](https://github.com/AlbertoPdRF) | [Lakshya Khatri](https://github.com/LakshyaKhatri) | [Saurabh Agarwala](https://github.com/SaurabhAgarwala) |
+| ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- |
```

Comment:
I'm not sure where all of this comes from, but if it looks good, then it's fine!

Reply:
It is done through auto-formatter. I check it, looks good